package.json

```diff
@@ -23,19 +23,22 @@
     "@polkadot-onboard/injected-wallets": "^0.0.2-beta",
     "@polkadot-onboard/react": "^0.0.2-beta",
     "@polkadot/api": "9.9.1",
+    "@polkadot/react-identicon": "^2.9.14",
     "@react-spring/web": "9.5.5",
     "framer-motion": "7.6.7",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-iconly": "2.2.5",
-    "react-remark": "2.1.0"
+    "react-remark": "2.1.0",
+    "styled-components": "^5.3.6"
```

Comment:
Have you considered using regular React `style` attribute?
In general I'd rather keep dependencies low unless there is a clear advantage.

Reply:
As far as I know, the style attribute is not recommended for major static styling.
https://reactjs.org/docs/dom-elements.html#style

 I like the [styled-components](https://styled-components.com/docs/basics) since it keeps the styles next to the components and avoids class name bugs and is also easy to migrate to sass or vanilla CSS if needed, but we should be able to switch to CSS modules or vanilla CSS, or any other css tool that is preferred.

Reply:
Good point. I guess at some points we will want to unify on something, I used `style` here and there.